src/main/webapp/script.js

```diff
@@ -262,17 +262,41 @@ class AutocompleteDirectionsHandler {
         }
     }
 
-    // Given A Route, by going through all of the legs, and each step inside each leg. 
-    scoreIndvRoute(Route, griddedData ) {
+    calculateAQI(lat, lng, griddedData) {
```

Comment:
```suggestion
    getAQI(lat, lng, griddedData) {
```

This isn't performing a calculation, simply acting as a getter for the griddedData.

Also; as noted by Rachel; there's a newly added function to do this in `master`; consider switching to that on the next rebase to reduce duplication.